appveyor.yml

```diff
@@ -1,4 +1,4 @@
-# This file was generated on 2019-01-03T20:34:23+00:00 from the rspec-dev repo.
+# This file was generated on 2019-01-08T11:28:05+00:00 from the rspec-dev repo.
```

Comment:
I think if you rebase this commit will go away

Reply:
I assume you are talking about the commit #2591 with sha 511cec5.  I am concerned that if I rebase this on my local machine and then push it, then the will be problems as I have already pushed this up. As well, if I look at rspec-core master, #2591 has a sha of cb1b4ce  If I do a diff between my local did_you_mean branch and my local master  it is not showing any files from 511cec5.  Frankly, I do not understand what is happening here.

Reply:
Its because its based off an older branch, github does that. You can use `git push <origin> <branch> --force-with-lease` to update this branch safely after a rebase, it won't let you ruin anything then.